components/Button.tsx

```diff
@@ -47,16 +43,25 @@ const StyledButton = styled(DEFAULT_TAG, {
   variants: {
     size: {
       '1': {
+        borderRadius: '$1',
```

Comment:
I really preferred the $2 border radius on the smaller buttons, I think it looks a bit more up to date. The inputs and ghost buttons still use $2, by the way.